sw/device/lib/testing/flash_ctrl_testutils.h

```diff
@@ -314,4 +314,21 @@ void flash_ctrl_testutils_data_region_print(
 void flash_ctrl_testutils_info_region_print(
     dif_flash_ctrl_info_region_t region, dif_flash_ctrl_region_properties_t *p,
     bool locked);
+// #endif  // OPENTITAN_SW_DEVICE_LIB_TESTING_FLASH_CTRL_TESTUTILS_H_
+
+/**
+ * Check for unlocked flash data regions
+ *
+ * This prints:
+ *
+ * @param flash_state A flash_ctrl state handle.
+ * @param start Start Address
+ * @param end End address
+ * @param region The info region descriptor.
+ * @param p The properties of the region.
+ * @param locked Whether or not the region is locked.
```

Comment:
Some of these param docs are stale I think:

```suggestion
 * @param flash A flash_ctrl state handle.
 * @param start Start of the region range to search.
 * @param end End of the region range to search.
 * @param region The data protection region.
```